pkg/config/loader.go

```diff
@@ -59,7 +59,7 @@ func LoadConfig() (*Config, error) {
 
 	// 3 - figure out what address we will tell pods to dial for aggregation
 	if cfg.Aggregation.AdvertiseAddress == "" {
-		if ip, ok := os.LookupEnv("SONOBUOY_ADVERTISE_IP"); ok {
+		if ip := os.Getenv("SONOBUOY_ADVERTISE_IP"); ip != "" {
```

Comment:
Yes, but I wouldn't mind going over how I verified to ensure I didn't miss anything.